src/bindings/jna/README.md

```diff
@@ -17,27 +20,48 @@ then you can compile and run [HelloElektra](HelloElektra.java):
 
     javac HelloElektra.java && java HelloElektra
 
-For plugin development, see [plugins](elektra/plugin)
+Alternatively you can also specify the classpath directly, both during compilation and execution. Note that in this example, we have the elektra and jna libraries in different directories, as mac doesn't allow writing to /usr/share/java in never versions. Also note its important in that case to include this directory containing the compiled HelloElektra.class when executing it, otherwise it will not be found:
+
+	javac -cp ./libelektra4j.jar:./libelektra4j/jna.jar HelloElektra.java
+	java -cp .:./libelektra4j.jar:./libelektra4j/jna.jar HelloElektra
+
+For plugin development, see [plugins](libelektra4j/plugin)
 and also [here](/src/plugins/jni) for more information.
 
+### Maven ###
+
+In case you want to use maven as your build system, simply go into the libelektra4j subfolder and execute mvn install. This will install the plugin into your local maven repository. Afterwards you can include it as a dependency into your maven project, and given that libelektra is actually installed on your system you can use it by including the maven dependency:
+	<groupId>org.libelektra</groupId>
+	<artifactId>libelektra4j</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
> But if there are no changes to the jna binding for a new elektra version, does it make sense to update the version number nevertheless?

In many Elektra releases also the C API has no changes, the number increments nevertheless. If there are changes in the C API we will add the new functionality in all bindings, too. So it makes sense to expose the version to the user. If you think we need an extra number to indicate incompatibilities of the bindings, we can add such a (single) number. But the easiest way is to use exactly the same number from the build system (use `KDB_VERSION` with `configure_file`, see `kdb.h.in`). It also would be consistent with the other bindings.

> Since it looks like some things are not really perfect yet with the jna bindings (see the readme) i'd use something like 0.9.0 or similar for now.

We should definitely avoid a number higher than Elektra's.